data/speakers.ts

```diff
@@ -57,14 +57,117 @@ const speakers: Speaker[] = [
+        videoUrl:
+          'https://www.youtube.com/watch?v=m3J-vOouaB4&index=17&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: {
+          continent: 'Europe',
+          country: 'Poland',
+          city: 'Wroclaw',
+        },
+      },
+      {
+        title: 'Building data driven mobile applications with React Native and GraphQL',
+        description: null,
+        eventName: 'DevExperience',
+        date: '2018-09-15',
+        videoUrl:
+          'https://www.youtube.com/watch?v=4Q1ElPWh2w8&index=12&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: {
+          continent: 'Europe',
+          country: 'Romania',
+          city: 'Iasi'
+        },
+      },
+      {
+        title: 'State of WebVR',
+        description: null,
+        eventName: 'DevTalks Romania',
+        date: '2018-09-18',
+        videoUrl:
+          'https://www.youtube.com/watch?v=54_iK9ruP7E&index=10&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: {
+          continent: 'Europe',
+          country: 'Romania',
+          city: 'Bucharest'
+        },
+      },
+      {
+        title: 'Building AR Apps with React Native',
+        description:'With the release of ARKit and ARCore by Apple and Google we see various Augmented reality apps created for iOS and Android. Have you ever wondered how you can create such apps in React Native? In this talk we will see how it can be done fairly easily.',
+        eventName: 'Byteconf',
+        date: '2018-08-31',
+        videoUrl:
+          'https://www.youtube.com/watch?v=1GToikXnNaw&index=7&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: null,
+      },
+      {
+        title: 'Building AR Apps with React Native',
+        description:'With the release of ARKit and ARCore by Apple and Google we see various Augmented reality apps created for iOS and Android. In this talk we will see how we can build AR apps with React Native, ARKit and ARCore.',
+        eventName: 'React Boston',
+        date: '2018-09-30',
+        videoUrl:
+          'https://www.youtube.com/watch?v=lN1DnHhludY&index=5&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: {
+          continent: 'North America',
+          country: 'USA',
+          city: 'Boston',
+        },
+      },
+      {
+        title: 'Reasonable GraphQL',
+        description:'ReasonML and GraphQL is a match made in heaven. Imagine getting type error if you have mistake in your GraphQL type even before compiling your client code. How cool is that? In this talk we will see how you can use GraphQL with Reason and what benefits it will bring.',
+        eventName: 'Byteconf',
+        date: '2018-11-30',
+        videoUrl:
+          'https://www.youtube.com/watch?v=wEjUjlZwbGU&index=3&list=PL28aKhmSneX8KsmkaSFFZWJCHqw4deSf5',
+        slidesUrl: null,
+        location: null,
```

Comment:
Hi @jediyozh, thanks for adding more talks!
Looks like this talk doesn't have any location because it was an online conference. `location: null` will break the page now, so I'll add the `online` option on FE and merge your PR after that.

Reply:
@yakovlevyuri Cool! Tell me when you finish and I'll update it :)

Reply:
Hi @yakovlevyuri! Do you have some updates?

Reply:
Hi, sorry for the late answer, didn't have any time to finish this :( Sorry for that.
Thanks for your PR, merged!

Reply:
@yakovlevyuri No problem, thank you👌